src/matchers/element/toHaveChildren.ts

```diff
@@ -1,45 +1,51 @@
 import { waitUntil, enhanceError, compareNumbers, numberError, executeCommand, wrapExpectedWithArray, updateElementsArray } from '../../utils'
 import { runExpect } from '../../util/expectAdapter'
 
-async function condition(el: WebdriverIO.Element, gte: number, lte: number, eq?: number): Promise<any> {
+async function condition(el: WebdriverIO.Element, options: ExpectWebdriverIO.NumberOptions): Promise<any> {
     const children = await el.$$('./*')
+
+    // If no options passed in + children exists
+    if(!options.lte && !options.gte && !options.eq) {
+        return {
+            result: (typeof children == 'undefined' || !children) ? false : true,
```

Comment:
Can you please simplify the condition?
```suggestion
            result: children != null,
```

Reply:
Yes nice, that is much nicer. I am going to see if I can write a unit test for this case as well tonight 👍  I'll push the change first though

Also do you think it is worth doing `children?.length` in case its null?

Reply:
Sounds good, can you please double check it?

Reply:
Added a test case for this and the optional chaining was needed so added it in. Also I noticed I had removed the 3rd param from the 3 matchers and so I added this back in because otherwise this kind of expect:

```
expect(el).toHaveChildren(5, {wait:5000})
```

Would have been broken so I added it back in! My bad

Was there anything else you want me to change?